GameServer/Logic/API/ApiFunctionManager.cs

```diff
@@ -103,18 +103,32 @@ public static void SetChampionModel(Champion champion, string model)
             {
                 var newCoords = _game.Map.AIMesh.getClosestTerrainExit(new Vector2(target.X, target.Y));
                 var newTarget = new Target(newCoords);
-                unit.DashTo(newTarget, dashSpeed, followTargetMaxDistance, backDistance, travelTime);
-                _game.PacketNotifier.notifyDash(unit, newTarget, dashSpeed, keepFacingLastDirection, leapHeight, followTargetMaxDistance, backDistance, travelTime);
+                unit.DashToTarget(newTarget, dashSpeed, followTargetMaxDistance, backDistance, travelTime);
+                _game.PacketNotifier.notifyDash(unit,
```

Comment:
This refers to other similar argument expansions.

For constructors what you're doing now is probably better, but maybe not for method calls.